lib/actions/api.js

```diff
@@ -615,16 +715,18 @@ export function getTransportationNetworkCompanyRideEstimate (params) {
 const receivedNearbyStopsResponse = createAction('NEARBY_STOPS_RESPONSE')
 const receivedNearbyStopsError = createAction('NEARBY_STOPS_ERROR')
 
-export function findNearbyStops (params) {
+export function findNearbyStops (params, parentStopId) {
```

Comment:
Can we rename `parentStopId` to `focusStopId` or something and also move away from using `childStops`? I feel like this is potentially confusing it with a parent stop in the GTFS sense where other stops are sub-stops of a parent stop.